app/src/main/java/org/oppia/app/topic/info/TopicInfoFragment.kt

```diff
@@ -12,6 +12,18 @@ import javax.inject.Inject
 
 /** Fragment that contains info of Topic. */
 class TopicInfoFragment : InjectableFragment() {
+  companion object{
+    /** Returns a new [TopicInfoFragment]. */
+    fun newInstance(internalProfileId: Int, topicId: String): TopicInfoFragment{
```

Comment:
Nit space issue:
`fun newInstance(internalProfileId: Int, topicId: String): TopicInfoFragment {`

Reply:
`{` after TopicInfoFragment should start from next line - is this what you meant?

Reply:
No there should be a space between { and TopicInfoFragment

Reply:
Also you can just ktlint -F filename to fix all these issues

Reply:
Done.

Reply:
Looks good. Thanks.